test/NameBinding/Inputs/NamedLazyMembers/NamedLazyMembers.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+
+@protocol doer
+
+- (void) doSomeWork;
+
+- (void) doSomeWorkWithSpeed:(int)s;
+
+- (void) doSomeWorkWithSpeed:(int)s andThoroughness:(int)t
+  NS_SWIFT_NAME(doSomeWork(speed:thoroughness:));
```

Comment:
Test style nitpick 3: Despite mockery, Cocoa naming almost never uses "and"; without it, the Swift name you've picked would match the usual selector-splitting rules Swift already applies. I suggest a test cases where you change the base name:

```objc
- (void)doSomeWorkWithSpeed:(int)s thoroughness:(int)t
  NS_SWIFT_NAME(doVeryImportantWork(speed:thoroughness:));
```

and one where you only change argument names

```objc
- (void)doSomeWorkWithSpeed:(int)s alacrity:(int)a
  NS_SWIFT_NAME(doSomeWork(speed:levelOfAlacrity:));
```

and check that both the "before" and "after" names get imported.

Reply:
No mockery intended! I'm just .. continuing to absorb ObjC via testcases and inspection of headers. Fixed.